controllers/chat/ChatController.js

```diff
@@ -446,7 +451,7 @@ module.exports = {
         error: validated
       });
     const client = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
-    const {members} = req.body;
+    const {members, oldChannelId, context = null} = req.body;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The destructuring of `req.body` has been updated to include `oldChannelId` and `context`. These new variables are used later in the code to handle anonymous-to-anonymous chat scenarios.
```diff
-    const {members} = req.body;
+    const {members, oldChannelId, context = null} = req.body;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->